docs/src/operate/security/certificate-signing-request.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Certificate signing request
+tags: [Operate, Security, Cloud]
+description: Generate certificate signing request for %%te%%
+---
+
+If you want to use a device certificate which is signed by a Certificate Authority (CA), you can generate the Certificate Signing Request (CSR), which is later used by CA to generate a device certificate. This process requires additional tooling as %%te%% provides you only with the CSR. 
+
+## Create a certificate signing request
+
+To create a CSR you can use [`tedge cert create-csr`](../../references/cli/tedge-cert.md) %%te%% command:
+
+```sh
+sudo tedge cert create-csr --device-id alpha
+```
+
+or
+
+```sh
+sudo tedge cert create-csr
+```
+if public key already exists and you want to reuse device name.
+
+```text title="Output"
+Certificate Signing Request was successfully created.
+```
+
+:::note
+`tedge cert` requires `sudo` privilege. This command provides no output on success.
+:::
+
+[`sudo tedge cert create-csr`](../../references/cli/tedge-cert.md) creates the certificate signing request in a default location (`/etc/tedge/device-certs/`).
+To use a custom location, refer to [`tedge config`](../../references/cli/tedge-config.md).
+
+Now you should have a CSR in the `/etc/tedge/device-certs/` directory.
+
+```sh
+ls -l /etc/tedge/device-certs/
+```
+
+```text title="Output"
+total 8
+-r--r--r-- 1 mosquitto mosquitto 664 May 31 09:26 tedge-csr.csr
```

Comment:
Unless we provide a `tedge` command to show the CSR, it would be good to provide the user a hint on how to check the CSR.

```
$ openssl req -in /etc/tedge/device-certs/tedge-csr.csr -noout -text
```